debian/changelog

```diff
@@ -1,3 +1,38 @@
+ubuntu-advantage-tools (27.14~23.04.1) lunar; urgency=medium
```

Comment:
@s-makin Please take at this draft of the changelog entry for our upcoming release.

Reply:
Will do!